src/ripple/app/paths/Pathfinder.cpp

```diff
@@ -157,6 +157,20 @@ smallestUsefulAmount(STAmount const& amount, int maxPaths)
 }
 }  // namespace
 
+template <class Stream>
+Stream&
+operator<<(Stream& s, Pathfinder::PathType const& v)
+{
+    auto iter = v.begin();
+    if (iter != v.end())
+    {
+        s << *iter;
+        for (++iter; iter != v.end(); ++iter)
+            s << ", " << *iter;
+    }
+    return s;
+}
```

Comment:
This kind of iterator comes up all the time - I wish someone would add it to boost or std. This is the best I could find:
https://en.cppreference.com/w/cpp/experimental/ostream_joiner
but that's experimental, I'd rather not use that.

Reply:
I couldn't help myself, so I wrote something to do it.